packages/fatfrog.lv2/PKGBUILD

```
@@ -0,0 +1,44 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=FatFrog
+pkgname=fatfrog.lv2
+pkgver=1.0
+pkgrel=3
+pkgdesc='A high gain valve amplifier simulation LV2 plugin'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name.lv2"
+license=(0BSD)
```

This should be changed to BSD, see [this comment](https://gitlab.archlinux.org/pacman/namcap/-/merge_requests/5#note_59140)
```suggestion
license=(BSD)
```

---

See https://github.com/osam-cologne/archlinux-proaudio/pull/143#discussion_r1012799603